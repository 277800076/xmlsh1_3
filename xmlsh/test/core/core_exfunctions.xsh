# Test of xmlsh xpath/ex functions
#
# Note this is an experimental feature
#
declare namespace xmlsh=java:org.xmlsh.xpath.XPathFunctions
#
xecho <[ xmlsh:eval("xecho $*" , ("foo" , <bar/> , 1 )  )  ]>
xquery -n 'xmlsh:eval("echo -n No Args")'
var=<[ xmlsh:eval("xecho <[ <foo>bar</foo> ]&gt; ") ]>
xecho $var
