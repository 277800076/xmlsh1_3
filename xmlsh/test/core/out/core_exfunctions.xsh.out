foo
<bar/>
1
No Args
<foo>bar</foo>
